docs/database/cosmosdb.md

```diff
@@ -61,9 +61,10 @@ For batch operations, if the individual operations are known to have the same op
 
 **[7]:** If using a port other than the default port for this DBMS and if `server.address` is set.
 
-**[8]:** For batch operations, if the individual operations are known to have the same query text then that query text SHOULD be used, otherwise all of the individual query texts SHOULD be concatenated with separator `; ` or some other database system specific separator if more applicable.
+**[8]:** Even though parameterized query text can potentially have sensitive data, by using a parameterized query the user is giving a strong signal that any sensitive data will be passed as parameter values, and the benefit to observability of capturing the static part of the query text by default outweighs the risk.
```

Comment:
this change seems off. We used to document what it should be be batch operations and now we talk about parametrization.

Perhaps we want both parts of this note to be present (here and in the registry)?

I have no objections addressing it here or moving to a separate PR.